src/plugins/lua/metadata_exporter.lua

```diff
@@ -63,6 +63,18 @@ Score: $score
 Symbols: $symbols]],
 }
 
+local variables = {
+  content = function(task)
+   return tostring(task:get_content())
```

Comment:
Please don't do it! `rspamd_text` is designed precisely to avoid doing tostring interning for large strings.

Reply:
Thx. I will check why I have added tostring here.